src/bmconfigparser.py

```diff
@@ -187,7 +188,7 @@ def read(self, filenames):
                     try:
                         if not self.validate(
                                 section, option,
-                                self[section][option]
+                                self[section][option]  # pylint: disable=unsubscriptable-object
```

Comment:
This one might be rewritten using `.get()`. Suppressing pylint warnings is a bad habit.